### PR TITLE
enable FEATURE_ENABLE_CLOSE_CONVERSATION feature flag in ROps in CI

### PR DESCRIPTION
**Pull request by davidmort**
# Motivation and Context
We need to set the FEATURE_ENABLE_CLOSE_CONVERSATION feature flag in CI so tests will pass

# What has changed
FEATURE_ENABLE_CLOSE_CONVERSATION set to True

### CODE REVIEW

**Review by shahi645 — APPROVED** [review]

File: pipelines/deployment.yml [hunk 1]
```diff
@@ -1327,6 +1327,7 @@ jobs:
         REDIS_SERVICE: ras-redis
         UAA_CLIENT_ID: 'response_operations'
         UAA_CLIENT_SECRET: ((ci_response_operations_client_secret))
+	FEATURE_ENABLE_CLOSE_CONVERSATION: 'True'
```
Comment (insacuri) [inline pipelines/deployment.yml:1330]: What's the red character doing there in the diff? Is that a tab character or something?. Should be fixed before it goes in.

**Review by insacuri — CHANGES_REQUESTED** [review]
See comment ^

File: pipelines/deployment.yml (hunk 1 above)
Comment (shahi645) [reply pipelines/deployment.yml:1330]: styling, he said hes going to fix

**Comment by gavinleeedwards** [comment]
The 'close conversation' feature has been live for months in production so rather than adding more config to set it in the pipeline, I've removed the feature flag from response-ops.  This PR can be closed.

**Pull request closed without merging**